core/templates/dev/head/profile/Preferences.js

```diff
@@ -35,6 +36,46 @@ oppia.controller('Preferences', ['$scope', '$http', '$rootScope', '$modal', '$ti
     _saveDataItem('user_bio', userBio);
   };
 
+  $scope.changedAtLeastOnce = false;
+  $scope.subjectInterestsWarningText = null;
+  $scope.TAG_REGEX = '^[a-z ]+$';
+
+  $scope.updateSubjectInterestsWarning = function(subjectInterests) {
+    var alpha = /^[A-Za-z]+$/;
```

Comment:
I don't understand... That needs to be fixed. We shouldn't have two different regexes for the same thing.